src/splat-debug.ts

```diff
@@ -72,11 +32,68 @@ class SplatDebug extends Element {
     }
 
     add() {
+        const vs = /* glsl */ `
+        attribute uint vertex_id;
+        attribute uint class_id;
+
+        uniform mat4 matrix_model;
+        uniform mat4 matrix_viewProjection;
+
+        uniform sampler2D splatState;
+        uniform highp usampler2D splatPosition;
+        uniform float splatSize;
+        uniform uvec2 texParams;
+
+        varying vec4 varying_color;
+
+        // Visualize class instances
+        uniform vec3 classColors[${GSplatLabels.MAX_UNIFORM_COLORS}];
```

Comment:
We are limited in the number of colors here, maybe there is a dynamic way of uploading the class colors?

Reply:
Probably better to sample the colors from a texture instead of uniform?